### PR TITLE
Move static-data/.gitignore in the root .gitignore.

### PR DESCRIPTION
**Pull request by XhmikosR**
Easier and better for maintenance to have everything in one place.

Also, clean up the entries.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
-/frontend/version
 /api/bulk_preloaded.go
```
Comment (lgarron) [inline .gitignore]: While we're at it, could we clean up and remove this entry?

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,4 @@
...
+/frontend/version
+/static-data/bulk-preloaded.json
+/static-data/dos.yaml
```
Comment (lgarron) [inline .gitignore]: This one, too!

**Comment by lgarron** [comment]
Thanks!

We can remove two of these, could you revise this to take them out?

**Comment by XhmikosR** [comment]
@lgarron: done!

**Comment by lgarron** [comment]
Thanks!

**Pull request merged**